apps/labs/posts/convoluted-story-behind-topk.md

```diff
@@ -0,0 +1,213 @@
+---
+title: 'The Convoluted Story Behind `np.top_k`'
+authors: [jules-poon]
+published: July 19, 2024
+description: 'In this blog post, I describe my experience as a first-time contributor to NumPy and talk about the story behind `np.top_k`.'
+category: [OSS Experience, Array API]
+featuredImage:
+  src: /posts/convoluted-story-behind-topk/NumPy_logo_2020_hero.webp
+  alt: 'NumPy Logo retrieved from Wikipedia Commons'
+hero:
+  imageSrc: /posts/convoluted-story-behind-topk/NumPy_logo_2020.webp
+  imageAlt: 'NumPy Logo retrieved from Wikipedia Commons'
+---
+
+In this blog post, I describe my experience as a first-time contributor and talk about the story behind `np.top_k`.
+
+---
+
+Hi~ I'm [Jules Poon](https://juliapoo.github.io/), a Year 2 undergraduate at the National University of Singapore. This summer I had the incredible opportunity to join the Open Source internship at Quansight Labs, under the mentorship of [Ralf Gommers](https://github.com/rgommers), [Nathan Goldbaum](https://ngoldbaum.github.io/) and [Matti Picus](https://github.com/mattip) for work on NumPy, and [Evgeni Burovski](https://github.com/ev-br) for work
```

Comment:
could you add "program" after "internship"?

Also, that home page of Nathan is pretty outdated - probably better to link https://github.com/ngoldbaum